crates/red_knot_workspace/resources/test/empty-unix-venv/bin/activate.bat

```diff
@@ -0,0 +1,59 @@
+@REM Copyright (c) 2020-202x The virtualenv developers
```

Comment:
We may want to remove files that aren't strictly necessary unless we plan to use the venv for other tests as well

Reply:
These activate scripts will never be relevant to any test we will ever write in red-knot, which is why I'd suggested to remove them.

Reply:
I don't have a strong objection to removing them, just that it feels like the more we "tamper" with the virtual environment, the less it feels like a "real" virtual environment and the more it feels like just a mock